### PR TITLE
Adding Rectangle (4:3) for cropping

### PR DESCRIPTION
**Pull request by Festanny**
Added a new Rectangle shape with a resolution of 4:3

### CODE REVIEW

File: Sources/SwiftyCrop/Models/CropViewModel.swift [hunk 1]
```diff
@@ -34,6 +43,17 @@ class CropViewModel: ObservableObject {
         let xLimit = ((imageSizeInView.width / 2) * scale) - maskRadius
         return CGPoint(x: xLimit, y: yLimit)
     }
+    func calculateDragGestureMaxRectangle() -> CGPoint {
```
Comment (benedom) [inline Sources/SwiftyCrop/Models/CropViewModel.swift:46]: Suggestion: Make this func private and use it in the above func `calculateDragGestureMax`. In the `CropViewModel` you should have access to the `maskShape`. Decide inside the func `calculateDragGestureMax` how to calculate based on the `maskShape`.

This also makes it easier below (I added a comment)

File: Sources/SwiftyCrop/View/CropView.swift [hunk 2]
```diff
@@ -50,7 +51,7 @@ struct CropView: View {
 
         let dragGesture = DragGesture()
             .onChanged { value in
-                let maxOffsetPoint = viewModel.calculateDragGestureMax()
+                let maxOffsetPoint = maskShape == .rectangle ? viewModel.calculateDragGestureMaxRectangle() : viewModel.calculateDragGestureMax()
```
Comment (benedom) [inline Sources/SwiftyCrop/View/CropView.swift:54]: (here)

If the func handles the mask shape you do not need to use a tenary here, makes it less complex and keeps logic inside the viewModel

**Review by benedom — CHANGES_REQUESTED** [review]

**Comment by benedom** [comment]
@Festanny I reworked the cropping functionality for rectangles, adding a more robust way and also making the aspect ratio configurable. Please review and test if this matches your requirements ☺️

**Review by benedom — APPROVED** [review]

**Comment by Festanny** [comment]
Yes, it fits, everything is cool!

**Pull request merged**